wrapper-kernel/jupyter_gap_wrapper/gap/setup.g

```diff
@@ -58,3 +58,6 @@ end);
 SetUserPreference("browse", "SelectHelpMatches", false);
 SetUserPreference("Pager", "cat");
 SetUserPreference("PagerOptions", "");
+
+# Display help in browser not a good option for servers
+# SetUserPreference( "HelpViewers", ["browser"])
```

Comment:
As @markuspf indicated, `SetUserPreference( "HelpViewers", ["browser"])` should be used on the user's side for those running Jupyter locally , not on the server. I suggest to revert these additon to `setup.g`.